runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/blog/BreadcrumbsSection/BreadcrumbsSection.module.scss

```diff
@@ -1,5 +1,49 @@
+@import '#src-landing/scss/main.scss';
+
+$SECTION-HEIGHT: 59px;
+$SEPARATOR_SPACE: $padding-1;
+
 .root {
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
     grid-column: span 2;
     width: 100%;
-    height: 59px;
+    height: $SECTION-HEIGHT;
+    margin-top: $padding-2;
+    margin-bottom: $padding-2;
+}
+
+.breadcrumb {
+    display: flex;
+}
+
+.link {
+    position: relative;
+    text-decoration: none;
+
+    &:hover::after {
+        content: '';
+        position: absolute;
+        bottom: -3px;
+        display: block;
+        width: 100%;
+        height: 2px;
+        background-color: $palette-accent-primary;
+    }
+}
+
+.homeLink {
+    display: flex;
+    align-items: center;
+    text-decoration: none;
+
+    .icon {
+        margin-right: $SEPARATOR_SPACE;
```

Comment:
make it padding. Prefer to apply padding instead of margins due to margins overlapping use them only if you want it like I did e.g. in  `RichTextRenderer`

Reply:
done